pkg/report/report.go

```diff
@@ -311,9 +311,10 @@ func Generate(ctx context.Context, path string, mrs yara.MatchRules, ignoreTags
 		ruleURL := generateRuleURL(m.Namespace, m.Rule)
 
 		b := &bincapz.Behavior{
-			RiskScore:    risk,
-			RiskLevel:    RiskLevels[risk],
+			Key:          key,
```

Comment:
nit: Can you rename this field ID?

IMHO, it's more important than just being a key; it's a unique ID for this behavwior.

Reply:
Addressed in [`290f13e` (#228)](https://github.com/chainguard-dev/bincapz/pull/228/commits/290f13e54bcde6059760f1807cfb98be377f024c).